models/uniswap/bnb/uniswap_bnb_sources.yml

```diff
@@ -0,0 +1,74 @@
+version: 2
+
+sources: 
+  - name: uniswap_v3_bnb
+    description: "BNB chain decoded tables related to Uniswap v3 contract"
+    freshness:
+      warn_after: { count: 12, period: hour }
+    tables:
+      - name: UniswapV3Pool_evt_Swap
```

Comment:
Seems this table cannot be found currently.

Reply:
I've submitted a UniswapV3Pool contract for decoding. However, since it's not been launched yet, it's not verified on BSCscan. I'm not sure if Dune can decode in this case. Also, we can wait for a few more days till Uniswap officially launched on BNB chain.
https://www.bscscan.com/address/0xbe429d8ea81443ca630f04cc79a5673b4cb587a7

Reply:
Should be fixed with latest commit
https://github.com/duneanalytics/spellbook/pull/2882/commits/ff0ccb70da1ef92a13fba95d3f154793b0fa4392